### PR TITLE
LRQA-81574 | Test Fix for ImportExport#CanDownloadSampleImportFile

### PR DESCRIPTION
**Pull request by Hijky**
Background:
Test Fix for ImportExport#CanDownloadSampleImportFile

Jira ticket:
https://liferay.atlassian.net/browse/LRQA-81574

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/dependencies/csv_account_import_sample.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
-customFields,description,domains,type,externalReferenceCode,parentAccountId,organizationIds,name,accountUserAccounts,status
-array,string,array,string,string,integer,array,string,array,integer
+customFields,description,domains,defaultShippingAddressId,type,defaultBillingAddressId,logoURL,postalAddresses,externalReferenceCode,parentAccountId,organizationIds,taxId,name,accountUserAccounts,logoId,status
```
Comment (magjed4289) [inline modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/dependencies/csv_account_import_sample.csv:1]: are we sure this is the expected result? Is it according to the acceptance criteria or clarified with PO?

Comment (Hijky) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/dependencies/csv_account_import_sample.csv:1]: I run the case ImportExport#CanDownloadSampleImportFile locally, and the download file is the same as the updated text

Comment (Hijky) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/dependencies/csv_account_import_sample.csv:1]: and I will run an RCA for it

Comment (Hijky) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/dependencies/csv_account_import_sample.csv:1]: Hey @magjed4289 I think we don't need to run RCA, I have found the [PR](https://github.com/brianchandotcom/liferay-portal/pull/96552/files) for it

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
The failures are not related to my PR

**Review by yanan — APPROVED** [review]

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/dependencies/csv_account_import_sample.csv (hunk 1 above)
Comment (magjed4289) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/dependencies/csv_account_import_sample.csv:1]: `I run the case ImportExport#CanDownloadSampleImportFile locally, and the download file is the same as the updated text`
yeah, that's for sure, but I'm referring to the fact whether the PMs are informed about this change and whether it is okay with the acceptance criteria.

**Review by magjed4289 — APPROVED** [review]

**Comment by Hijky** [comment]
ci:forward

**Comment by Hijky** [comment]
Manually forward the PR to Brian https://github.com/brianchandotcom/liferay-portal/pull/138044

**Pull request closed without merging**